src/ert/config/ensemble_config.py

```diff
@@ -51,28 +48,6 @@ def _get_abs_path(file: Optional[str]) -> Optional[str]:
     return file
 
 
-@dataclass(eq=False)
-class Refcase:
```

Comment:
Any reason to move this?

Reply:
`EnsembleConfig` imports `ResponseConfig`, and `ResponseConfig` now also imports `Refcase` due to the observation parsing being there, so it would be circular if `Refcase` was moved to `EnsembleConfig`.